src/Client/MultiplexedConnections.cpp

```diff
@@ -119,6 +119,12 @@ void MultiplexedConnections::sendQuery(
 
     Settings modified_settings = settings;
 
+    /// Consistent write to log_queries across replicas
```

Comment:
It needs to be the same as for main query.
So, if the main query is logged, the remote queries should also.